src/lwc/geFormWidgetTokenizeCard/__tests__/geFormWidgetTokenizeCard.test.js

```diff
@@ -128,6 +145,94 @@ describe('c-ge-form-widget-tokenize-card', () => {
             expect(spanDisabledMessage(element).innerHTML).toBe(BDI_FAILURE_DISABLED_MESSAGE);
         });
     });
+
+    it('should not display ACH input fields when in batch mode', async () => {
+        const element = createWidgetElement();
+        setPaymentMethod(element, ACH);
+        element.widgetDataFromState = {
+            ...element.widgetDataFromState,
+            [DATA_IMPORT_PARENT_BATCH_LOOKUP]: 'DUMMY_ID'
+        }
+        document.body.appendChild(element);
+
+        return Promise.resolve()
+            .then(() => {
+                expect(iframe(element)).toBeNull();
+                expect(spanExtendedDisabledMessage(element).innerHTML).toBe(EXTENDED_DISABLED_MESSAGE);
+                expect(doNotEnterPaymentButton(element)).toBeFalsy();
+            });
+    });
+
+    it('should go into hard read-only mode when credit card payment has been captured', async () => {
+        const element = createWidgetElement();
+        element.paymentTransactionStatusValues = {
+            AUTHORIZED: 'AUTHORIZED',
+            CAPTURED: 'CAPTURED'
+        }
+        element.widgetDataFromState = {
+            ...element.widgetDataFromState,
+            [PAYMENT_METHOD_FIELD]: CREDIT_CARD,
+            [DATA_IMPORT_PARENT_BATCH_LOOKUP]: 'DUMMY_ID',
+            [DATA_IMPORT_PAYMENT_STATUS]: 'CAPTURED'
+        };
+
+        getObjectInfoAdapter.emit(mockObjectInfo);
```

Comment:
```suggestion
        getObjectInfo.emit(mockObjectInfo);
```
Can emit directly to the imported wire adapter. Check for other references to the (now deleted) testAdapters, and use the adapter directly.